docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/FastTree.cs

```diff
@@ -45,11 +75,21 @@ private static IEnumerable<DataPoint> GenerateRandomDataPoints(int count)
             }
         }
 
+        // Example with label and 50 feature values. A data set is a collection of such examples.
         private class DataPoint
         {
             public float Label { get; set; }
             [VectorType(50)]
             public float[] Features { get; set; }
         }
```

Comment:
I assume all the samples are going to use same DataPoints (to be consistent), right?

Reply:
I guess so. We now have samples, examples, instances, which are kind of less precise than data point.

Reply:
the label type and number of features might change depending on the scenario, but the outline is the same.

---
In reply to: [266160360](https://github.com/dotnet/machinelearning/pull/2979#discussion_r266160360) [](ancestors = 266160360)